### ISSUE TITLE
[Bug] bot not regonizing player

### ISSUE THREAD

**Issue by Trop3x** [SDClowen/RSBot#694]
**Version (please complete the following information):**
 - SRO version: [Rigid v1.228]
 - RSBot version (or commit): [2.9.1.0]

**Describe the bug**
I start client with bot-->logging in manually--> select char-->start---> when i go to check bot, i cant find my char in player list. It's empty (No Selected). Only thing i can do with bot is hide/show client.

**Comment by toma619** [comment]
yes me too same Problem i playing ExaySro i dont know which Sro should i Chose ?
